packages/wallet-adapter-react/src/useWallet.tsx

```diff
@@ -35,15 +36,15 @@ export interface WalletContextState {
   wallets: ReadonlyArray<Wallet>;
   signAndSubmitTransaction<T extends Types.TransactionPayload, V>(
     transaction: T,
-    options?: V
+    options?: TransactionOptions
   ): Promise<any>;
   signAndSubmitBCSTransaction<T extends TxnBuilderTypes.TransactionPayload, V>(
     transaction: T,
-    options?: V
+    options?: TransactionOptions
   ): Promise<any>;
   signTransaction<T extends Types.TransactionPayload, V>(
     transaction: T,
-    options?: V
+    options?: TransactionOptions
   ): Promise<any>;
   signMessage(message: SignMessagePayload): Promise<SignMessageResponse | null>;
   signMessageAndVerify(message: SignMessagePayload): Promise<boolean>;
```

Comment:
Can remove `V` now, I don't think it's used for anything, or V should be TransactionOptions

Reply:
yea.. we either use `TransactionOptions` or `V extends TransactionOptions`, I would say the later

Reply:
Thank you for your valuable suggestions. commit  [1281dba](https://github.com/aptos-labs/aptos-wallet-adapter/pull/156/commits/1281dba03db8c0e8140c1339573a27fd636c287a)